frontend/test/metabase/scenarios/sharing/alert/alert.cy.spec.js

```diff
@@ -1,13 +1,16 @@
 import {
   restore,
-  setupLocalHostEmail,
+  setupDummySMTP,
```

Comment:
I need to completely remove this from all tests.
It's not a recommended way to set up an email and it doesn't even hit the email endpoint.

It probably doesn't matter for this case because you just need app to think that there is some email config present. However, I'd use real maildev settings.

Sorry for not removing this helper earlier. It'll be on my plate first thing tomorrow.